tests/functional/services/auth/base/test_get_identity_providers.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+import globus_sdk
+from globus_sdk._testing import get_last_request, load_response
+
+
+def test_get_identity_providers(client):
```

Comment:
I'd like to see a test that verifies we can get IDPs by query parameters as well. I'm recommending renaming this so there's symmetry across the test names.

```suggestion
def test_get_identity_providers_by_domains(client):
```

Reply:
I'm reopening because I'd like to see test(s) that exercise passing query parameters.

Reply:
I'll get that added in two shakes of a dog's tail.

Reply:
R(er)esolving this with the test now in place.